test/bigtest/interactors/title-create.js

```diff
@@ -25,12 +26,12 @@ import {
   fillName = fillable('[data-test-eholdings-title-name-field] input');
 
   hasContributorBtn = isPresent('[data-test-eholdings-contributors-fields] [data-test-repeatable-field-add-item-button]');
-  addContributor(type, name) {
-    return this
-      .click('[data-test-eholdings-contributors-fields] [data-test-repeatable-field-add-item-button]')
-      .fill('[data-test-eholdings-contributor-type] select', type)
-      .fill('[data-test-eholdings-contributor-contributor] input', name);
-  }
+  clickAddContributor = clickable('[data-test-eholdings-contributors-fields] [data-test-repeatable-field-add-item-button]');
+  contributorsRowList = collection('[data-test-eholdings-contributors-fields] li', {
+    type: fillable('[data-test-eholdings-contributor-type] select'),
+    contributor: fillable('[data-test-eholdings-contributor-contributor] input'),
```

Comment:
maybe it's better to name them like enterType and enterContributor?

Reply:
data-test-eholdings-contributor-contributor seems redundant but I can't think of a better name to suggest